dev/user.clj

```diff
@@ -1,18 +1,23 @@
 (ns user
   (:require [figwheel-sidecar.repl-api]))
 
+
 (defn start-server! []
   (figwheel-sidecar.repl-api/start-figwheel!
     (assoc-in (figwheel-sidecar.config/fetch-config)
               [:data :figwheel-options :server-port] 4541)
     "dev-server")
   (figwheel-sidecar.repl-api/cljs-repl "dev-server"))
 
-(defn start-ui! []
-  (figwheel-sidecar.repl-api/start-figwheel!
-    (figwheel-sidecar.config/fetch-config)
-    "dev")
-  (figwheel-sidecar.repl-api/cljs-repl "dev"))
+
+(defn start-ui!
```

Comment:
swap the variable in config without defining another build, see my comment below.